test/domain_errors.jl

```diff
@@ -22,13 +22,13 @@ end
     M = ErrorTestManifold()
     @test isa(check_point(M, [-1, 1]), DomainError)
     @test check_point(M, [1, 1]) === nothing
-    @test !is_point(M, [-1, 1])
-    @test is_point(M, [1, 1])
-    @test_throws DomainError is_point(M, [-1, 1], true)
+    @test !is_point(M, [-1.0, 1.0])
+    @test is_point(M, [1.0, 1.0])
+    @test_throws DomainError is_point(M, [-1.0, 1.0], true)
 
     @test isa(check_vector(M, [1, 1], [-1, 1]), DomainError)
     @test check_vector(M, [1, 1], [1, 1]) === nothing
-    @test !is_vector(M, [1, 1], [-1, 1])
-    @test is_vector(M, [1, 1], [1, 1])
-    @test_throws DomainError is_vector(M, [1, 1], [-1, 1], true)
+    @test !is_vector(M, [1.0, 1.0], [-1.0, 1.0])
+    @test is_vector(M, [1.0, 1.0], [1.0, 1.0])
+    @test_throws DomainError is_vector(M, [1.0, 1.0], [-1.0, 1.0], true)
```

Comment:
With my change to `is_vector` and `is_point` this should work with integers.

Reply:
ah, I was lazy and assumed, that these are just not good types anymore then. I will add an integer test then to ensure that this is further doable.